extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/mongodb/runtime/LiquibaseMongodbBuildTimeConfig.java

```diff
@@ -3,25 +3,26 @@
 import java.util.List;
 import java.util.Optional;
 
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
 
 /**
  * The liquibase configuration
  */
-@ConfigRoot(name = "liquibase-mongodb", phase = ConfigPhase.BUILD_TIME)
-public class LiquibaseMongodbBuildTimeConfig {
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
```

Comment:
Why changing to be available in runtime too? Better keep the original comfig

Reply:
Because it's actually used at runtime.

I initially tried to move it to the deployment artifact and I couldn't (and you get an error from the new layer because I suppose it's a bit stricter).

Reply:
See https://github.com/quarkusio/quarkus/pull/46080/files#diff-02d52ffa86efe275669766110547b55691a603fb066360ffde5f19b251d30429R55